src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -154,12 +154,13 @@ def skimToDataTier(cmsswVersion, skim):
         command += "cd %s\n" % scramBaseDirs[0]
         command += "eval `scramv1 runtime -sh`\n"
 
-        command += """python -c 'from Configuration.StandardSequences.Skims_cff import getSkimDataTier\n"""
```

Comment:
and here we could keep the dual compatibility by
```
if PY3:
   ### create the python3 command
else:
  ### create the python2 command
```

Reply:
Or actually, something like:
```
        whichPython = "python3" if PY3 else "python"
        command += """%s -c 'from Configuration.StandardSequences.Skims_cff import getSkimDataTier\n""".format(whichPython)
        command += """dataTier = getSkimDataTier("%s")\n""" % skim
        command += """if dataTier:\n\tprint(dataTier.value())'"""
```
should be enough. Could you please give this a try?

Reply:
First option works, but using `whichPython = "python3" if PY3 else "python"` fails again with the same issue.

Reply:
Hmm, maybe because we would have to decode that string to bytes(?)
Anyhow, sorry for giving you another suggestion, but I think this code can be enhanced with something like:
```
        if PY3:
            command += """python3 -c 'from Configuration.StandardSequences.Skims_cff import getSkimDataTier\n"""
        else:
            command += """python -c 'from Configuration.StandardSequences.Skims_cff import getSkimDataTier\n"""
        command += """dataTier = getSkimDataTier("{}")\n""".format(skim)
        command += """if dataTier:\n\tprint(dataTier.value())'"""
```
Can you please provide these changes before running a replay to test it? Just to make sure you got the correct formatting in place. Thanks

Reply:
@jhonatanamado Jhonatan, you are missing the white space between `print` and `datatier`, which has been provided in the discussion above. Can you please fix it?

Reply:
Ups sorry! I will fixed asap